corehq/apps/app_manager/static/app_manager/js/forms/advanced/case_properties.js

```diff
@@ -1,7 +1,12 @@
-"use strict";
-hqDefine('app_manager/js/forms/advanced/case_properties', function () {
-    var caseConfigUtils = hqImport('app_manager/js/case_config_utils');
-
+hqDefine("app_manager/js/forms/advanced/case_properties", [
+    "knockout",
+    "underscore",
+    "app_manager/js/case_config_utils",
```

Comment:
The case_config_utils module is included a couple of lines above this module (case_properties): https://github.com/dimagi/commcare-hq/blob/5fe14801267a27d02a155896edf9171583f0cba9/corehq/apps/app_manager/templates/app_manager/form_view.html#L26-L30